### ISSUE TITLE
Aggregating directory data on multi-network

### ISSUE THREAD

**Issue by danaskallman** [meitar/multisite-directory#16]
I was wondering what the options are in terms of aggregating directory data across networks on a multi-network. Meaning can other multisite in the same multi-network display for example affiliated members. 

A specific use case I have would be for the national coop organization to also display the regional coops in the multi-network. So parent network is the national organization providing the infrastructure and a multisite for each regional network. For example there is a NY network of coops, and Madison and Austin, etc. They all have a multisite in the national multi-network. Each region displays a list of their regions coops and the national would display an aggregated list of all regions/multisites on the multi-network. 

Does this make sense, or is it confusing?

**Comment by fabacab** [comment]
Yeah, so, a `[network-directory]` much like the existing `[site-directory]` but to display info about the Multisites that exist rather than the individual sites, yes? (I.e., same general idea but one level up?)

**Comment by danaskallman** [comment]
Yeah, that's it in a nutshell. Though I am wondering how this affects a Taxonomy across the networks? If at all.

**Comment by danaskallman** [comment]
A thought, it would be good to have an option for sites to op-out of the directory. 

The idea is that in most cases site 1 or parent site of the multi-site would display the directory and that site shouldn't be displayed. But I can see the case where I want to display that main site site of each multi-site only on the main network site to show for example all the regional networks that are part of the coalition. 

Adding thoughts as they come up.
